packages/terra-scroll/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Added
   * Added styles when the scroll is focussed
+* Changed
+  * Removed the focus style changes added for terra-scroll.
```

Comment:
(update `a.b.c` to the appropriate version)
```suggestion
* Changed
  * Reverted the focus style changes added in `a.b.c`.
```

Reply:
Updated